nav2_util/include/nav2_util/odom_subscriber.hpp

```diff
@@ -32,20 +32,21 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef NAV_2D_UTILS__ODOM_SUBSCRIBER_HPP_
-#define NAV_2D_UTILS__ODOM_SUBSCRIBER_HPP_
+#ifndef NAV2_UTIL__ODOM_SUBSCRIBER_HPP_
```

Comment:
We have the OdomSmoother in Nav2 Utils, shouldn't we use that now instead to give more accurate results?